docs/overview/threat-model.md

```diff
@@ -14,21 +14,28 @@ The Backstage trust model is divided into three groups with different trust leve
 
 An **internal user** is an authenticated user that generally belongs to the organization of a particular Backstage deployment. These users are trusted to the extent that they are not expected to compromise the availability of Backstage, but they are not trusted to not compromise data confidentiality or integrity.
 
-An **integrator** is a user responsible for configuring and maintaining an instance of Backstage. Integrators are fully trusted, since they operate the system and database and therefore have root access to the host system. Additional measures can be taken by adopters of Backstage in order to restrict or observe the access of this group, but that falls outside of the current scope of Backstage.
+An **operator** is a user responsible for configuring and maintaining an instance of Backstage. Operators are fully trusted, since they operate the system and database and therefore have root access to the host system. Additional measures can be taken by adopters of Backstage in order to restrict or observe the access of this group, but that falls outside of the current scope of Backstage.
```

Comment:
I guess we should follow up and remove all other integrator mentions in other parts of the repo. Some day.

Reply:
Yep wanna settle that BEP first, but for the threat model I felt it was clearer to make the update already